CSSSSupportLibrary/SupportLibrary/OSVersionInfo/WinNTVersionInfo.OSVersionInfo.SupportLibrary.cs

```diff
@@ -61,13 +61,13 @@ public enum ProcessorArchitecture
         /// <summary>
         /// Determines if the current application is 32 or 64-bit.
         /// </summary>
-        static public SoftwareArchitecture ProgramBits
+        public static SoftwareArchitecture ProgramBits

```

Comment:
I'm not sure if changes can be made to anything in this file - it's based off [https://www.codeproject.com/Articles/73000/Getting-Operating-System-Version-Info-Even-for-Win](https://www.codeproject.com/Articles/73000/Getting-Operating-System-Version-Info-Even-for-Win), but I don't know the exact wording of their license. Could you either check to see if modifications can be made, or revert the changes?

Reply:
[The licence](https://www.codeproject.com/info/cpol10.aspx) states that `You may apply bug fixes, portability fixes and other modifications obtained from the Public Domain or from the Author. A Work modified in such a way shall still be considered the standard version and will be subject to this License.` I feel this would be sufficient under other modifications as it is conforming with the style requirements of the project. It is neither adding or removing large functional parts.

Reply:
Thanks for finding that information out for me, so I'm happy for the changes to be made.

The license also states `You may otherwise modify Your copy of this Work (excluding the Articles) in any way to create a Derivative Work, provided that You insert a prominent notice in each changed file stating how, when and where You changed that file.` in the point under the one you've quoted.

I think adding a brief comment about the changes, the date you made them and a link to this PR so all of the changes can be seen will be sufficient.